Mathlib/SetTheory/Ordinal/FixedPoint.lean

```diff
@@ -5,6 +5,7 @@ Authors: Violeta Hernández Palacios, Mario Carneiro
 -/
 import Mathlib.SetTheory.Ordinal.Arithmetic
 import Mathlib.SetTheory.Ordinal.Exponential
+import Mathlib.Logic.UnivLE
```

Comment:
Wait, the `UnivLE` API can be used to construct a `Small.{max u v, u}` instance? I was under the impression this just didn't work, if it does then that's awesome!